nevergrad/optimization/optimizerlib.py

```diff
@@ -2673,6 +2673,17 @@ def _select_optimizer_cls(self) -> base.OptCls:
             return super()._select_optimizer_cls()
 
 
+@registry.register
+class NGOpt17(NGOpt16):
+    def _select_optimizer_cls(self) -> base.OptCls:
+        if (
+            self.budget is not None and self.budget > 500 * self.dimension and self.fully_continuous
+        ):  # Discrete case ?
+            return ConfPortfolio(optimizers=[NGOpt14] * 9, warmup_ratio=0.7)
+        else:
+            return super()._select_optimizer_cls()
+
+
 @registry.register
 class NGOpt(NGOpt16):
```

Comment:
do you keep NGOpt to 16?

Reply:
Let's switch to 17, given the cool results.